doc/user/content/get-started/quickstart.md

```diff
@@ -447,66 +436,21 @@ Let's see this in action.
     (2 rows)
     ```
 
-1. Once the `r2` replica is ready (`ready=t`), drop the `r1` replica to simulate a failure:
-
-    ```sql
-    DROP CLUSTER REPLICA compute_qck.r1;
-    ```
-
-1. Query the indexed view that you created in `compute_qck` earlier:
-
-    ```sql
-    SELECT * FROM avg_bids;
-    ```
-    <p></p>
-
-    ```nofmt
-            item        |    average_bid
-    --------------------+--------------------
-     Custom Art         | 49.770776201263864
-     Gift Basket        |   49.8909070204407
-     City Bar Crawl     | 50.056635368698046
-     Best Pizza in Town |  50.50023551577956
-     Signed Memorabilia |  50.11854192264935
-    (5 rows)
-
-    Time: 23.537 ms
-    ```
-
-    As you can see, the results are available despite the failure of one of the cluster's replicas.
+    When the new replica shows `ready=t`, it has caught up with the original replica. From that moment forward, they are logical clones that do the same computation and hold the same results.
 
 ## Step 6. Scale up or down
 
-In addition to using replicas to increase fault tolerance, you can add and remove replicas to scale resources up or down according to the needs of a cluster. For example, let's say the `2xsmall` replica in the `compute_qck` cluster is running low on memory.
-
-1. Add the next largest replica, `xsmall`:
-
-    ```sql
-    CREATE CLUSTER REPLICA compute_qck.r3 SIZE = 'xsmall';
-    ```
+You can scale the physical resources of a cluster up or down based on need. For example, let's say `2xsmall` is not providing enough memory for the work of the `compute_qck` cluster.
 
-1. Use the [`SHOW CLUSTER REPLICAS`](https://materialize.com/docs/sql/show-cluster-replicas/) command to check the status of the new replica:
+1. Use the `ALTER CLUSTER` command to bump the cluster up to the next largest size, `xsmall`:
 
     ```sql
-    SHOW CLUSTER REPLICAS WHERE CLUSTER = 'compute_qck';
+    ALTER CLUSTER compute_qck SET (SIZE 'xsmall');
     ```
-    <p></p>
 
-    ```noftm
-       cluster   | replica |  size   | ready
-    -------------+---------+---------+-------
-     compute_qck | r2      | 2xsmall | t
-     compute_qck | r3      | xsmall  | t
-    (2 rows)
-    ```
-
-1. Once the `r3` replica is ready (`ready=t`), it's safe to remove the `r2` replica:
-
-    ```sql
-    DROP CLUSTER REPLICA compute_qck.r2;
-    ```
+    Behind the scenes, this command adds a new `xsmall` replica, catches the replica up to the previous `2xsmall` replica, and then removes the `2xsmall` replica.
```

Comment:
```suggestion
    Behind the scenes, this command adds a new `xsmall` replica and removes the `2xsmall` replica. There may be an interruption in service while the new replica catches up to where the old replica left off.
```

Reply:
PTAL again, @benesch.

Reply:
The important bit here is that at this time, changes are applied instantaneously, meaning that we don't wait for a new instance to catch up before tearing down the old replicas. This is subject to change in the future where we want to be smarter about it.

Reply:
LGTM!